### PR TITLE
nixos/input-remapper: Fix missing [], add more options, add test

### PR DESCRIPTION
**Pull request by LunNova**
###### Motivation for this change
Fixes a bug, adds two new options for the service WantedBy target and the udev rules, and adds a test that the module works.

Disables python unit tests for the package due to flaky tests under high load. https://github.com/sezanzeb/input-remapper/issues/306

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by timokau** [comment]
Unfortunately I don't have the capacity to review this right now.
/status needs_reviewer

File: nixos/tests/input-remapper.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          ./common/x11.nix
+        ];
+
+        services.xserver.enable = true;
+        services.input-remapper.enable = true;
+        users.users.sybil = { isNormalUser = true; group = "wheel"; };
+        test-support.displayManager.auto.user = user.name;
+        # passwordless pkexec bodge
+        security.sudo = { enable = true; wheelNeedsPassword = false; };
+        security.wrappers.pkexec = pkgs.lib.mkForce
```
[the remark this answers is not in the document]
Comment (timokau) [reply nixos/tests/input-remapper.nix:23]: Unfortunately I don't know a better way to fix this. Could you maybe add a comment that explains the situation?

**Review by timokau — CHANGES_REQUESTED** [review]
Thanks! Looks good, just one small nit (documenting the `pkexec` issue). Sorry for the delay.

File: nixos/tests/input-remapper.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (LunNova) [reply nixos/tests/input-remapper.nix:30]: Added:

```
        # workaround for pkexec not working in the test environment
        # Error creating textual authentication agent:
        #   Error opening current controlling terminal for the process (`/dev/tty'):
        #   No such device or address
        # passwordless pkexec with polkit module also doesn't work
        # to allow the program to run, we replace pkexec with sudo
        # and turn on passwordless sudo
        # this is not correct in general but good enough for this test
```

**Review by timokau — APPROVED** [review]

**Pull request merged**